src/lib/operators/aggregate_hash.hpp

```diff
@@ -95,18 +83,14 @@ using DistinctColumnType = int8_t;
 using DistinctAggregateType = int8_t;
 
 /**
- * Note: Aggregate does not support null values at the moment
+ * Note: AggregateHash does not support null values at the moment
```

Comment:
I believe we should remove this line (markus said the hash aggregate should support null values atm, and there are actually (green) tests for null values)